tofu/utils.py

```diff
@@ -639,7 +647,7 @@ def _get_exception(q, ids, qtype='quantity'):
 
 
 def load_from_imas(shot=None, run=None, user=None, tokamak=None, version=None,
-                   ids=None, Name=None, out=None, tlim=None, config=None,
+                   ids=None, Name=None, returnas=None, tlim=None, config=None,
```

Comment:
out keyword replaced by returnas for clarity (will do it gradually everytime I see an opportunity)